// @test error

fn main() : int
{
    int x = x + 1;
    return x;
}
